pkg/workspace/workspace_queries.go

```diff
@@ -66,24 +66,36 @@ func (w *Workspace) ResolveQueryAndArgsFromSQLString(sqlString string) (*modconf
 	}
 
 	// 2) is this a file
-	fileQuery, fileExists, err := w.getQueryFromFile(sqlString)
+	// get absolute filename
+	filePath, err := filepath.Abs(sqlString)
+	if err != nil {
+		return nil, nil, fmt.Errorf("error opening file '%s': %s", sqlString, err.Error())
+	}
+	fileQuery, fileExists, err := w.getQueryFromFile(filePath)
 	if fileExists {
 		if err != nil {
-			return nil, nil, fmt.Errorf("ResolveQueryAndArgsFromSQLString failed: error opening file '%s': %v", sqlString, err)
+			return nil, nil, fmt.Errorf("error opening file '%s': %s", filePath, err.Error())
 		}
 		if fileQuery == nil {
-			error_helpers.ShowWarning(fmt.Sprintf("file '%s' does not contain any data", sqlString))
+			error_helpers.ShowWarning(fmt.Sprintf("file '%s' does not contain any data", filePath))
 			// (just return the empty query - it will be filtered above)
 		}
 		return fileQuery, nil, nil
 	}
+	// sqlString is not a file name that exists
```

Comment:
`// sqlString is not a file name that exists` => `// the argument cannot be resolved as an existing file`